src/questions.js

```diff
@@ -0,0 +1,156 @@
+const starterQuestions = [
+  {
+    type: "input",
```

Comment:
Consider adding validation to all of your questions. We can leave all of these blank currently - this isn't handled in your generate readme code, so I end up with a README with lots of blanks